release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -733,6 +759,13 @@ the imagestream has the pull-through policy enabled. In this case, an
 `ImagePullBackOff` error displays.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1787112[*BZ#1787112*])
 
+* The Che Workspace Operator was updated to remove support for the Workspace custom resource and add support for the DevWorkspace custom resource. However, the OpenShift web terminal continues to use the Workspace custom resource instead of the DevWorkspace custom resource for starting the terminal workspaces. Hence it fails to work with the latest version of the Che Workspace Operator. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1846969[*BZ#1846969*])
```

Comment:
This one was a little confusing to me. I suggest rewording this. Here is one possibility:

"The Che Workspace Operator was updated to use the DevWorkspace custome resource instead of the Workspace custom resource. However, the OpenShift web terminal continues to use the Workspace custom resource. Because of this, the OpenShift web terminal fails to work with the latest version of the Che Workspace Operator. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1846969[*BZ#1846969*])"

Reply:
Fixed